metadata-io/src/main/java/com/linkedin/metadata/entity/restoreindices/RestoreIndicesArgs.java

```diff
@@ -0,0 +1,64 @@
+package com.linkedin.metadata.entity.restoreindices;
+
+public class RestoreIndicesArgs implements Cloneable {
+    public int start = 0;
+    public int batchSize = 10;
+    public int numThreads = 1;
+    public long batchDelayMs = 1;
+    public String aspectName;
+    public String urn;
+    public String urnLike;
+
+    @Override
+    public RestoreIndicesArgs clone() {
+        try {
+            RestoreIndicesArgs clone = (RestoreIndicesArgs) super.clone();
+            // TODO: copy mutable state here, so the clone can't change the internals of the original
+            return clone;
+        } catch (CloneNotSupportedException e) {
+            throw new AssertionError();
+        }
+    }
+
+    public RestoreIndicesArgs setAspectName(String aspectName) {
```

Comment:
Can just use "@Data" from lombok

Reply:
That will cause return type to be void. I wanted a builder pattern for ease of use.